charts/ocis/values.yaml

```diff
@@ -85,20 +85,22 @@ insecure:
   ocisHttpApiInsecure: false
 
 cache:
-  # -- Type of the cache to use. It defaults to no cache or a in memory cache depending on the service.
+  # -- Type of the cache to use. It defaults to no cache.
   # To disable the cache, set to "noop".
-  # Can be set to "redis", "redis-sentinel" or "etcd", then the address of Redis (Sentinel) / etcd node(s) needs to be set to `cache.nodes`.
-  type: ""
+  # Can be set to "redis-sentinel", "etcd" or "nats-js-kv" then the address of Redis (Sentinel) / etcd / nats-js node(s) needs to be set to `cache.nodes`.
+  type: "nats-js-kv"
   # -- Nodes of the cache to use.
-  nodes: []
-  # nodes:
-  #   - redis-master-1.ocis-redis.svc.cluster.local:6379
-  #   - redis-master-2.ocis-redis.svc.cluster.local:6379
+  nodes:
+    - nats.ocis-nats.svc.cluster.local:4222
```

Comment:
please do it like for the store where the nodes are set to `- "{{ .appNameNats }}:9233"`

the current approach only works if deployed to the `ocis-nats` namespace

Reply:
Did it like suggested. We should think about a way to read variables from `messagingSystem.external` if enabled.

Reply:
It crashes locally when using `{{ .appNameNats }}` it works better with `nats`. Are we sure that the templates are executed when defining them in the `values.yml`?

Reply:
There is some extra step that templates it:

```
            - name: CLIENTLOG_STORE_NODES
              value: {{ tpl (join "," .store.nodes) . | quote }}
```

Reply:
I get error messages like
```
"Failed to connect to NATS Server: parse \"nats://{{ .appNameNats }}:9233\": invalid character \"{\" in host name"
```
when using `- "{{ .appNameNats }}:9233"` it works when using `- "nats:9233"`

Are we sure that the template is resolved correctly?